src/commands/watch/watch.ts

```diff
@@ -28,17 +25,24 @@ export class Watch implements Command {
     const options: WatchOptions = {};
 
     this.args.forEach((arg) => {
-      if (arg.includes("--entryPoint=")) { // TODO Improve this code
-        const entryPoint = arg.split("=")[1];
-        options.entryPoint = entryPoint;
-      } else if (arg.includes("--cwd=")) {
-        const cwd = arg.split("=")[1];
-        options.cwd = cwd;
-      } else if (arg.includes("--runtimeOptions")) {
-        const runtimeOption = arg.split("=")[1].split(",");
-        options.runtimeOptions = runtimeOption;
-      } else if (arg.includes("--help")) {
+      if (/^--help/.test(arg) || /^-h/.test(arg)) {
         options.help = true;
+      } else if (/^--.+=/.test(arg)) {
+        let m = arg.match(/^--([^=]+)=([\s\S]*)$/);
```

Comment:
This line could be replaced by `const [, key, value] = arg.match(/^--([^=]+)=([\s\S]*)$/);`

Reply:
I'm getting the fallowing error, I looked for solutions but I couldn't find one, do you have any idea what it is?
I'm not very familiar with TypeScript, I'm still learning

`Type 'RegExpMatchArray | null' must have a '[Symbol.iterator]()' method that returns an iterator. ts(2488)`

Reply:
Yes my bad, match can return `null` so can you just rename `m` to something like `option` and put it in a const?
To create `key` and `value` variables you can replace
```
let key = option[1];
let value = option[2];
```
by something like that
```
const [,key, value] = option;
```
So I suggest something like that:

```
const option = arg.match(/^--([^=]+)=([\s\S]*)$/);
if (option === null) return;
const [,key, value] = option;
```

Reply:
Sure, it's done.
Thanks for your help.